docs/projects/structure/next-js-16-usage.md

```
@@ -0,0 +1,378 @@
+- **User Experience:** "Internal Server Error" 500 responses
+- **Production Logs:**
+  ```
+  [Error: An error occurred in the Server Components render...] {
+    digest: 'DYNAMIC_SERVER_USAGE'
+  }
+  ```
+
+**Phase 2 - Build Error (export const dynamic):**
+
+- **Build Output:**
+  ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)